input/fsh/SD_Radiotherapy.fsh

```diff
@@ -98,35 +99,36 @@ Description: "Extension capturing a technique of external beam or brachytherapy
 * insert ExtensionContext(Procedure)
 * value[x] only CodeableConcept
 
-Extension: RadiotherapyDose
+Extension: RadiotherapyFractionsDelivered
+Id:        mcode-radiotherapy-fractions-delivered
+Title:     "Radiotherapy Fractions Delivered"
+Description: "The total number of fractions (treatment divisions) actually delivered for this volume."
+* insert ExtensionContext(Procedure)
+* value[x] only unsignedInt
+
+Extension: RadiotherapyDoseDelivered
 Id: mcode-radiotherapy-dose
 Title: "Radiotherapy Dose"
 Description: "Dose parameters for one target volume, including dose per fraction, number of fractions delivered, and total dose delivered."
 * insert ExtensionContext(Procedure)
 * extension contains
-    targetVolumeDescription 0..1 and
-    targetVolumeId 0..1 and
-    dosePerFraction 0..1 and
-    deliveredFractions 0..1 and
+    volumeDescription 0..1 and
+    volumeId 0..1 and
+//    dosePerFraction 0..1 and
```

Comment:
Do you want to explain why these lines are commented out or delete them?

Same with a few lines below this.